tests/test_metrics.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+
+import numpy as np
+from scipy.stats import ortho_group
+from sklearn.datasets import load_iris
+
+from skcosmo.metrics import (
+    global_reconstruction_error,
+    global_reconstruction_distortion,
+    local_reconstruction_error,
+)
+
+
+class ReconstructionMeasuresTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        features = load_iris().data
+        cls.features_small = features[:20, [0, 1]]
+        cls.features_large = features[:20, [0, 1, 0, 1]]
+        cls.eps = 1e-5
+        cls.n_local_points = 15
+
+        np.random.seed(0x5F3759DF)
+        cls.features_rotated_small = cls.features_small.dot(
+            ortho_group.rvs(cls.features_small.shape[1])
+        )
+
+    def test_global_reconstruction_error_identity(self):
```

Comment:
needs doc

Reply:
While the tests have meaningful names, they don't explain what the tests do. I'll add some suggestions for doc
```suggestion
    def test_global_reconstruction_error_identity(self):
    # tests that the GRE of a dataset on itself returns within a threshold of zero
```

Reply:
This is a representative for all review comments about adding docs to the two-liner tests:

But you understood this from the code instantly, the code is self-explanatory. I don't think one should be forced to add doc to such simple code. Especially it is just repetitive. I will add the suggestions you send to it, because you already put the work of writing them. But I dont think one should be required to add doc to such simple code.